src/styles/index.ts

```diff
@@ -1,3 +1,31 @@
-export { default as breakpoints } from './breakpoints';
-export { default as colors } from './colors';
-export { fonts, fontSizes } from './fonts';
+import styled from 'styled-components/macro';
+
+import breakpoints from './breakpoints';
+import colors from './colors';
+import { fonts, fontSizes } from './fonts';
+
+export const NotMobileOnly = styled.div`
+  display: none;
+
+  @media ${breakpoints.notMobile} {
+    display: initial;
+  }
+`;
+
+export const NotTabletOnly = styled.div`
+  display: none;
+
+  @media ${breakpoints.notTablet} {
+    display: initial;
+  }
+`;
+
+export const TabletOnly = styled.div`
+  display: none;
+
+  @media ${breakpoints.tablet} {
+    display: initial;
+  }
+`;
```

Comment:
Invert the breakpoints to avoid unnecessary `display: initial` override

Reply:
There are cases where this pattern makes the most sense. Let's keep this unless you have suggestions for a workaround that doesn't use this pattern, and is equally as clean.

Reply:
```
export const NotTabletOnly = styled.div`
  @media ${breakpoints.tablet} {
    display: none;
  }
`;

export const TabletOnly = styled.div`
  @media ${breakpoints.notTablet} {
    display: none;
  }
`;
```

Also, `display: initial` can potentially break elements that are `display: flex`/`display: grid`

Reply:
Ah I see, I think I misunderstood your initial comment.

Reply:
You're saying that `display: initial` on a parent element can break child elements that are `display: flex` or `display: grid`? That doesn't seem right, can you give an example?

Reply:
I think I meant, if you ever wanted to override `display: initial` as currently implemented it wouldn't be as easy because you'd also have to include the breakpoint to avoid clashing with the default `display: none`.